portal-bridge/src/utils.rs

```diff
@@ -69,6 +71,36 @@ pub fn read_test_assets_from_file(test_path: PathBuf) -> TestAssets {
 
     assets
 }
+/// Gets the duration until the next light client update
+/// Updates are scheduled for 4 seconds into each slot
+pub fn duration_until_next_update(genesis_time: u64) -> Duration {
+    let current_slot = expected_current_slot(genesis_time);
+    let next_slot = current_slot + 1;
+    let next_slot_timestamp = slot_timestamp(next_slot, genesis_time);
+
+    let now = SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .expect("Time went backwards")
```

Comment:
"Timestamp overflow" is probably the more likely scenario, though I suppose it would happen if someone set their computer time to 1969, too.

It's not super urgent obviously, but as far as I understand, if someone runs trin on a computer with 32-bit timestamps in 2038, this will panic.